.github/workflows/build.yml

```diff
@@ -47,6 +42,18 @@ jobs:
             echo "VERSION=${{ github.sha }}" >> $GITHUB_ENV
           fi
 
+      - name: Update VERSION file
+        if: ${{ startsWith(github.ref, 'refs/tags/v') }}
+        working-directory: thingy91x-oob
+        run: |
+          python3 scripts/app_version.py ${GITHUB_REF_NAME} > app/VERSION
```

Comment:
I considered doing that, but even though both are dealing with versions they're not doing the same thing. `Set VERSION` is exposing a string as an environment variable, but `Update VERSION file` is modifying a file and doesn't depend on `Set VERSION`. So I figured it would be cleaner to have them separately. Is there a down-side to have them separately?

Reply:
No down side really. Just wanted to not repeat code. Since they are not doing the same thing, leave it as is